documentation/api/query/v4/catalogs.markdown

```diff
@@ -47,6 +47,7 @@ See [the AST query language page][ast].
   recent catalog.
 * `producer_timestamp` (string): a string representing the time at which the
   `replace_catalog` command for a given catalog was submitted from the master.
+* `producer` (string): the certname of the Puppet master that sends the catalog to PuppetDB.
```

Comment:
you could even replicate the passive voice in the line above, but I don't feel as strongly there. The sends just jumped out at me.